mux/mux.go

```diff
@@ -3,104 +3,87 @@ package mux
 import (
 	"context"
 	"errors"
-	"net"
+	"log"
 	"net/http"
+	"strings"
 	"time"
 
-	"github.com/soheilhy/cmux"
+	"golang.org/x/net/http2"
+	"golang.org/x/net/http2/h2c"
 	"google.golang.org/grpc"
 )
 
 const defaultGracePeriod = 5 * time.Second
+const grpcRequestMarker = "application/grpc"
```

Comment:
how about 
```go

const (
	defaultGracePeriod = 5 * time.Second
	grpcRequestMarker = "application/grpc"
)
```

Reply:
yes. missed when i copy pasted the graceperiod part from entropy. will fix this.